bin/rucio-conveyor-stager

```diff
@@ -1,15 +1,21 @@
 #!/usr/bin/env python
-# Copyright European Organization for Nuclear Research (CERN)
+# Copyright 2015-2018 CERN for the benefit of the ATLAS collaboration.
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
-# You may not use this file except in compliance with the License.
-# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 #
 # Authors:
-# - Vincent Garonne, <vincent.garonne@cern.ch>, 2012
-# - Mario Lassnig, <mario.lassnig@cern.ch>, 2013-2015
-# - Cedric Serfon, <cedric.serfon@cern.ch>, 2013
```

Comment:
Why are some persons removed again from the header?

Reply:
this one only comes from the gitlog. this removes e.g., the person who wrote code in another file and that was then copy/pasted by someone else. this one for example is my startup code below, but it was copypasted by wen from the conveyor.

Reply:
Ok, I think in general this is fine, we should just be careful about removing people from the credits who maybe worked on something together or debugged this together but only one person made the commit.

Reply:
I assume it is fine like that